app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor.js

```diff
@@ -157,12 +157,12 @@ export default function Editor({
           label={label}
           description={description}
           options={/** @type {string[]} */ (property.enum || [])}
-          isEnum={true}
```

Comment:
I think this should not be changed. Making it possible to delete the values is achieved with setting `disableClearable` to `false` below and the (false) red highlighting is removed with the change in `DropdownAttribute`.
As far as I can see, this change only opens up the possibility to input invalid values.

Reply:
When leaving this enabled `null` is used as a value after clearing. As a result the field is never removed.

Reply:
I see. In that case there's no difference between `STRING_ENUM` and `STRING_WITH_OPTIONS` except for where the `options` come from. We could unify those again. But I'm also fine with leaving it as it is right now. Your call.

Reply:
I would leave it as it is for now. I think we will replace the dropdown component in the future.